govcd/entity_test.go

```diff
@@ -71,6 +71,9 @@ func (vapp *VApp) id() string   { return vapp.VApp.ID }
 func (vm *VM) name() string { return vm.VM.Name }
 func (vm *VM) id() string   { return vm.VM.ID }
 
+func (disk *Disk) name() string { return disk.Disk.Name }
```

Comment:
If we don't use `testFinderGetGenericEntity` for Disk, implementing the `genericEntityInterface` is not needed

Reply:
removed